apps/volontulo/views/pages.py

```diff
@@ -12,19 +14,24 @@
 from apps.volontulo.models import Page
 
 
-# pylint: disable=too-many-ancestors;missing-docstring
+# pylint: disable=too-many-ancestors,missing-docstring
 class PageList(ListView):
     model = Page
     template_name = 'pages/page_list.html'
 
+    @method_decorator(user_passes_test(
+            lambda u: u.is_authenticated() and u.userprofile.is_administrator))
+    def dispatch(self, *args, **kwargs):
```

Comment:
Maybe it's a little bit too strict but our pylint configuration right now requires docstrings for every method.